azure-pipelines.yml

```diff
@@ -46,8 +46,8 @@ stages:
         set -ex
         sudo apt-get -y purge libhiredis-dev libnl-3-dev libnl-route-3-dev
         sudo dpkg -i ../target/debs/buster/{libswsscommon_1.0.0_amd64.deb,python3-swsscommon_1.0.0_amd64.deb,libnl-3-200_*.deb,libnl-genl-3-200_*.deb,libnl-nf-3-200_*.deb,libnl-route-3-200_*.deb,libhiredis0.14_*.deb}
-        sudo python3 -m pip install ../target/python-wheels/swsssdk*-py3-*.whl
-        sudo python3 -m pip install ../target/python-wheels/sonic_py_common-1.0-py3-none-any.whl
+        sudo python3 -m pip install ../target/python-wheels/buster/swsssdk*-py3-*.whl
```

Comment:
>buster [](http://example.com/codeflow?start=60&length=6)

What is the reason that they are using different Debian version?
I think we should stick to the version same as whatever used in docker-snmp image.

Reply:
Fixed it, snmp is using buster debian version.